ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/ccj/CCDRepaymentPlan.java

```diff
@@ -10,7 +10,6 @@
 @Builder
 public class CCDRepaymentPlan {
 
-    private BigDecimal firstPayment;
```

Comment:
This might need to be removed from CCD spreadsheet.

Reply:
it does

Reply:
Thanks Sabah for making change in spreadsheet as part of your changes. I have tested it locally after spreadsheet changes